sql/data_model.sql

```diff
@@ -0,0 +1,64 @@
+BEGIN;
+
+-- add pgcrypto extension so that we can use the gen_random_uuid function.
+CREATE EXTENSION IF NOT EXISTS pgcrypto;
+
+DROP TABLE IF EXISTS formdata CASCADE;
+CREATE TABLE formdata(
+  id uuid PRIMARY KEY DEFAULT gen_random_uuid(),
+  domain text NOT NULL,
+  received_on timestamp without time zone,
+  time_start timestamp without time zone,
+  time_end timestamp without time zone,
+  duration bigint,
+  device_id text,
+  user_id uuid NOT NULL,
+  username text,
+  app_id uuid,
+  xmlns text NOT NULL,
+  form_json text NOT NULL
+);
+
+CREATE INDEX formdata_domain on formdata(domain);
+
+DROP TABLE IF EXISTS casedata CASCADE;
+CREATE TABLE casedata(
+  id uuid PRIMARY KEY DEFAULT gen_random_uuid(),
+  domain text NOT NULL,
+  version text,
+  type text,
+  closed boolean DEFAULT FALSE,
+  user_id uuid,
```

Comment:
they're the same. it's pretty confusing that `user_id` is used to represent the person who last submitted against a case so i was just proposing changing it. then again, it's also pretty ingrained in the product / codebase that that's what's happening so maybe better not to change.

as a third consideration, we could just remove it since i don't think it should be required for the standalone processor.